Orion.Net/Controllers/BaseDataController.cs

```diff
@@ -1,14 +1,13 @@
-﻿using Microsoft.AspNetCore.Authorization;
+﻿using System;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Newtonsoft.Json;
 using Orion.Net.Core.Interfaces;
 using StackExchange.Redis;
-using System;
-using System.Collections.Generic;
 
 namespace Orion.Net.Controllers
 {
-
+    [Authorize]
```

Comment:
Proposal :
Can you replace it by the [Authorize(Policy = "PostAPI")] directly ?
If you add [Authorize] the client must be authenticated with OAuth

Reply:
It blocks the support (No Authorization) and the client still needs to be authenticated : Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectHandler: Information: AuthenticationScheme: AzureADOpenID was challenged.

Reply:
And if we open everything with [AllowAnonymous], then specify [Authorize(Policy = "PostAPI")] : it doesn't work. [AllowAnonymous] bypass every specifications below it.